src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime() ?: ''
+        if(image != "") {
+            def url = Utils.shWithOutput(this, "git config remote.origin.url")
```

Comment:
Perhaps we should have a Utils method for getting the remote url
cc @hrishin @piyush-garg